src/sys/socket/mod.rs

```diff
@@ -70,6 +69,22 @@ bitflags!(
     }
 );
 
+// Flags for send/recv and their relatives
+libc_bitflags!{
+    pub flags MsgFlags: libc::c_int {
+        MSG_OOB,
```

Comment:
Can you add doc comments for each of these? They're documented [here](http://man7.org/linux/man-pages/man2/send.2.html).